src/common/Selects/PledgeFrequencySelect.tsx

```diff
@@ -0,0 +1,26 @@
+import { MenuItem, MenuItemProps, Select, SelectProps } from '@mui/material';
+import { PledgeFrequencyEnum } from 'src/graphql/types.generated';
+
+type PledgeFrequencySelectProps = Partial<SelectProps> & {
+  MenuItemProps?: Partial<MenuItemProps> & { [testid: string]: string };
+  getFunctionLabel: (label: string) => React.ReactNode;
```

Comment:
Because we don't need to customize the label, we can use `getLocalizedPledgeFrequency` directly instead of passing in `getFunctionLabel` as a prop.